package.json

```diff
@@ -78,6 +77,7 @@
 		"optional-require": "1.0.3"
 	},
 	"devDependencies": {
+		"@sap-devx/app-studio-toolkit-types": "file:./types",
```

Comment:
This is an indicator we need to transform this repo into a mono-repo.
- **Separate** topic of tech debt.